integrations/cv.js

```diff
@@ -0,0 +1,289 @@
+// Helpers
```

Comment:
Any way we can import this without having to copy/paste?

Reply:
I'm not sure if there's an elegant way to. Closest I can think of would be just periodically fetch and store it from GitHub but I have to imagine it would be changing so little that would be more effort than it's worth versus just updating it whenever it gets updated in the SDK